ocp_resources/virtual_machine_instance.py

```diff
@@ -31,7 +35,7 @@ def __init__(
         client=None,
         privileged_client=None,
         yaml_file=None,
-        delete_timeout=TIMEOUT_4MINUTES,
+        delete_timeout=TIMEOUT_8MINUTES,
```

Comment:
This now changes the default behavior for anyone using the wrapper; if this is needed for tests, it should be done there.

Reply:
But we determined 8 mins to be a reasonable timeout for VMI deletion. Was there logic to the previous default timeout  except for 'this is what we encountered by this point'?

Reply:
it is reasonable for cnv internal usage; default should not be modified.

Reply:
Ok. I'll revert